src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  background-color: $background-color-main;
+  margin: 0;
+}
+
+.app {
+  text-align: center;
+}
+
+.active {
+  color: $background-color-purple !important;
+  border-bottom: 4px solid $background-color-purple;
+}
+
+.players {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+  font-family: Abadi;
+  height: 45px;
+  box-shadow: 1px -1px 2px #D4D3D3;
+
+  .team {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    position: relative;
+    background: $background-color-purple;
+
+    .logo-container {
+      display: flex;
+      background-color: white;
+      border-radius: 24px;
+      box-shadow: 1px 1px 5px $border-dark-grey;
+      margin: 0 15px;
+    }
+
+    button {
+      background: none;
+      color: inherit;
+      border: none;
+      padding: 0;
+      cursor: pointer;
+      outline: inherit;
+      color: white;
+      text-transform: uppercase;
+      font-size: 22px;
+      font-family: spurs;
+      margin: 0 15px 0 0;
+
+      i {
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 10px;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 99;
+      border-top: 45px solid #D0073E;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .links {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1.5;
+    background-color: white;
+    font-family: Avenir;
+    text-transform: uppercase;
+    font-size: 16px;
+  
+    .link-container {
+      display: inline-block;
+      height: 100%;
+      margin: 0 25px;
+    }
+  
+    a {
+      color: black;
+      display: inline-block;
+      height: calc(100% - 19px);    
+      padding-top: 15px;
+      text-decoration: none;
+  
+      &:hover {
+        height: calc(100% - 19px);
+        color: $grey-three;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 99;
+      border-top: 45px solid white;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .user {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: $background-color-grey;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    color: $text-dark-grey;
+    padding-left: 15px;
+  
+    div {
+      display: flex;
+      margin: 0 20px;
+    }
+
+    button {
+      display: flex;
+      align-items: center;
+      background: none;
+      color: inherit;
+      border: none;
+      padding: 0;
+      cursor: pointer;
+      outline: inherit;
+      font-size: 14px;
+      margin: 0 15px 0 0;
```

Comment:
instead of margin, add a padding so that the button is clickable

`padding-left: 15px;`